Solutions/Infoblox Cloud Data Connector/Analytic Rules/Infoblox-TI-CommonSecurityLogMatchFound-MalwareC2.yaml

```diff
@@ -0,0 +1,68 @@
+id: 5b0864a9-4577-4087-b9fa-de3e14a8a999
+name: Infoblox - TI - CommonSecurityLog Match Found - MalwareC2
+description: |
+  'CommonSecurityLog (CEF) MalwareC2/MalwareC2DGA match found in your Infoblox TIDE Threat Intelligence. Customize query count, scheduling, responses and more. Modify data sources, types and threat properties as desired.'
+severity: Medium
+status: Available
+requiredDataConnectors:
+  - connectorId: CEF
+    dataTypes: 
+      - CommonSecurityLog
+  - connectorId: ThreatIntelligence
+    dataTypes: 
+      - ThreatIntelligenceIndicator
+queryFrequency: 1h
+queryPeriod: 14d
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - Impact
+relevantTechniques:
+  - T1498
+  - T1565
+query: |
+  let dt_lookBack = 1h;
+  let ioc_lookBack = 14d;
+  let TI = ThreatIntelligenceIndicator
+  | where TimeGenerated >= ago(ioc_lookBack)
+  //Get most recent ingested indicator in case there are copies
```

Comment:
Hi @sschuur can you please remove the comment from between these lines, the comment is being read as a separate line by the KQL validation tests and is resulting in failure.